x/wasmplus/keeper/api.go

```diff
@@ -0,0 +1,100 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/line/lbm-sdk/types"
+	wasmkeeper "github.com/line/wasmd/x/wasm/keeper"
+	"github.com/line/wasmd/x/wasm/types"
+	wasmplustypes "github.com/line/wasmd/x/wasmplus/types"
+	wasmvm "github.com/line/wasmvm"
+)
+
+type cosmwasmAPIImpl struct {
+	keeper *Keeper
+	ctx    *sdk.Context
+}
+
+func (a cosmwasmAPIImpl) callCallablePoint(contractAddrStr string, name []byte, args []byte, isReadonly bool, callstack []byte, gasLimit uint64) ([]byte, uint64, error) {
+	contractAddr := sdk.MustAccAddressFromBech32(contractAddrStr)
+	contractInfo, codeInfo, prefixStore, err := a.keeper.ContractInstance(*a.ctx, contractAddr)
+	if err != nil {
+		return nil, 0, err
+	}
+
+	if a.keeper.IsInactiveContract(*a.ctx, contractAddr) {
+		return nil, 0, fmt.Errorf("called contract cannot be executed")
+	}
+
+	env := types.NewEnv(*a.ctx, contractAddr)
+	wasmStore := wasmplustypes.NewWasmStore(prefixStore)
+	gasRegister := a.keeper.GetGasRegister()
+	querier := wasmkeeper.NewQueryHandler(*a.ctx, a.keeper.GetWasmVMQueryHandler(), contractAddr, gasRegister)
+	gasMeter := a.keeper.GasMeter(*a.ctx)
+	api := a.keeper.CosmwasmAPI(*a.ctx)
+
+	instantiateCost := gasRegister.ToWasmVMGas(gasRegister.InstantiateContractCosts(a.keeper.IsPinnedCode(*a.ctx, contractInfo.CodeID), len(args)))
+	if gasLimit < instantiateCost {
+		return nil, 0, fmt.Errorf("lack of gas for calling callable point")
+	}
+	wasmGasLimit := gasLimit - instantiateCost
+
+	result, events, attrs, gas, err := a.keeper.GetWasmVM().CallCallablePoint(name, codeInfo.CodeHash, isReadonly, callstack, env, args, wasmStore, api, querier, gasMeter, wasmGasLimit, wasmkeeper.CostJSONDeserialization)
+	gas += instantiateCost
+
+	if err != nil {
+		return nil, gas, err
+	}
```

Comment:
```suggestion
	result, events, attrs, gas, err := a.keeper.GetWasmVM().CallCallablePoint(name, codeInfo.CodeHash, isReadonly, callstack, env, args, wasmStore, api, querier, gasMeter, wasmGasLimit, wasmkeeper.CostJSONDeserialization)
	gas += instantiateCost
	if err != nil {
		return nil, gas, err
	}
```

It is not recommended to add a newline if it is a semantic syntactic unit(chunk).